src/System.ComponentModel.EventBasedAsync/src/System/ComponentModel/BackgroundWorker.cs

```diff
@@ -23,6 +23,11 @@ public BackgroundWorker()
             _operationCompleted = new SendOrPostCallback(AsyncOperationCompleted);
             _progressReporter = new SendOrPostCallback(ProgressReporter);
         }
+        
+        ~BackgroundWorker()
```

Comment:
Can you add a comment here explaining that this finalizer exists for compatibility purposes, with a link to this pull request?